Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -4585,8 +4586,12 @@ void WebPage::updatePreferences(const WebPreferencesStore& store)
 
     // FIXME: This should be automated by adding a new field in WebPreferences*.yaml
     // that indicates override state for Lockdown mode. https://webkit.org/b/233100.
-    if (WebProcess::singleton().isLockdownModeEnabled())
+    if (WebProcess::singleton().isLockdownModeEnabled()) {
         adjustSettingsForLockdownMode(settings, store);
+        // FIXME. Add the domain name?
+        // FIXME. Add RELEASE_LOG too?
+        m_page->console().addMessage(MessageSource::Security, MessageLevel::Info, makeString("This device has activated Lockdown Mode. It will affect some functionalities on the Web page."));
```

Comment:
I'm not sure this is really working as intended yet. It needs to be double checked.

Reply:
@brentfulgham guidance would be welcome here.

Reply:
I doubt the console logging will be helpful here. My bet is that you'll want to move this check / logging to `WebLocalFrameLoaderClient::dispatchDidFinishLoad()` when `m_frame->isMainFrame()`. This way, it would lot once in the main frame after the load completes.

Reply:
I removed it for now.